src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html

```diff
@@ -106,7 +106,20 @@
 {% endif %}
 
 {% if invoice_email_enabled and has_invoices %}
-    <form method="post" action="{% eventurl event "plugins:banktransfer:mail_invoice" order=order.code secret=order.secret %}">
+    {% if info.send_invoice_to %}
+        <p>
+            {% blocktrans trimmed with recipient=info.send_invoice_to %}
+                At your request, we are sending the invoice directly to {{ recipient }}.
```

Comment:
Usually when this message is displayed, the invoice was already sent (only exception I can think of is if it's still in the task queue for actually sending the email), right? So we should better use past tense here

Reply:
Yes. I was worried about the task not yet executed or the email not sent successfully, but let's not bother the custoemr with it.